blazor-features/BlazorWebAssemblyCustomEvents/BlazorWebAssemblyCustomEvents/CustomEvents/EventHandlers.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Microsoft.AspNetCore.Components;
+using Microsoft.AspNetCore.Components.Forms;
+
+namespace BlazorWebAssemblyCustomEvents.CustomEvents
+{
+    [EventHandler("ondoubleclick", typeof(DoubleClickEventArgs), enableStopPropagation: true, enablePreventDefault: true)]
+    public static class EventHandlers
+    {
+        // purposefully empty
+    }
+
+    public class DoubleClickEventArgs : EventArgs
+    {
+        public DateTime? EventTimestamp { get; set; }

```

Comment:
Is there a reason why the `EventTimestamp` property is nullable? I think that we always need to have the `EventTimestamp`